### PR TITLE
docs: Small fixes to the Resource deletion section of API Concept

### PR DESCRIPTION
**Pull request by austince**
- Finalizers are a string array
- Small grammar error

### CODE REVIEW

**Comment by sftim** [comment]
/sig api-machinery

LGTM for docs

**Comment by liggitt** [comment]
Lgtm for technical content

File: content/en/docs/reference/using-api/api-concepts.md [hunk 1]
```diff
@@ -724,13 +724,13 @@ When you **delete** a resource this takes place in two phases.
...
   "metadata": {
-    "finalizers": {"url.io/neat-finalization", "other-url.io/my-finalizer"},
+    "finalizers": ["url.io/neat-finalization", "other-url.io/my-finalizer"],
     "deletionTimestamp": nil,
   }
 }
 ```
 
-When a client first sends a **delete** to request removal of a resource, the `.metadata.deletionTimestamp` is set to the current time.
+When a client first sends a **delete** to request the removal of a resource, the `.metadata.deletionTimestamp` is set to the current time.
```
Comment (aj11anuj) [inline content/en/docs/reference/using-api/api-concepts.md:733]:
```suggestion
When a client first sends a **delete** request for the removal of a resource, then the `.metadata.deletionTimestamp` is set to the current time.
```

Comment (sftim) [reply content/en/docs/reference/using-api/api-concepts.md:733]: Hi @aj11anuj 

The suggestion you've made may be unhelpful; the change proposed in the PR is more idiomatic (at least as UK English).

**Comment by austince** [comment]
@sftim @liggit thanks for the reviews? What's needed to merge?

**Comment by sftim** [comment]
Thanks

/lgtm
/approve

Tech LGTM: https://github.com/kubernetes/website/pull/43689#issuecomment-1780967912

**Pull request merged**